protocols/smtp/src/main/java/org/apache/james/protocols/smtp/core/UnknownCmdHandler.java

```diff
@@ -47,6 +48,8 @@
      * The name of the command handled by the command handler
      */
     private static final Collection<String> COMMANDS = ImmutableSet.of(UnknownCommandHandler.COMMAND_IDENTIFIER);
+    private static final String MISSING_CURR_COMMAND = "";
+    public static final ProtocolSession.AttachmentKey<String> CURR_COMMAND = ProtocolSession.AttachmentKey.of("CURR_COMMAND", String.class);
```

Comment:
Can't we put all SMTP attachment keys in a common place?

Reply:
some AK are only use locally